panther_ioc_rules/sunburst_fqdn_iocs.py

```diff
@@ -0,0 +1,15 @@
+from panther_iocs import sunburst_fqdn_ioc_match, intersection, SUNBURST_FQDN_IOCS
+
+
+def rule(event):
+    return sunburst_fqdn_ioc_match(event)
+
+
+def title(event):
+    title_str = "Sunburst Indicator of Compromise Detected"
+    if "p_any_domain_names" in event:
```

Comment:
At this point we know that `p_any_domain_names` must be in `event`, otherwise we'd never have gotten to this point right? I'd remove this `if` statement, I think it might be confusing about because it might lead you to wonder how you could get here if `p_any_domain_names` is not present.